color/src/color.rs

```diff
@@ -442,7 +459,7 @@ impl<CS: ColorSpace> PremulColor<CS> {
 
     /// Convert a color to the corresponding separate alpha form.
     #[must_use]
-    pub fn un_premultiply(self) -> AlphaColor<CS> {
+    pub const fn un_premultiply(self) -> AlphaColor<CS> {
```

Comment:
:+1:

We might look for other opportunities to add `const`. In this case, it's a gimme, as `premultiply` is also const, so I would say that not having it was an oversight, corrected by this PR.